src/main/lib/utils.js

```diff
@@ -1,8 +1,12 @@
 import path from 'path';
 
 export function getPathToFile(fileName) {
```

Comment:
Sorry - this should be `getURIPathToFile`, as it uses the `file:/` uri.

Reply:
k